src/configure-webpack/index.js

```diff
@@ -83,6 +83,16 @@ const buildSharedWebpackConfig = (saguiConfig) => {
       })
     ],
 
+    // In wepback 2 builtin node.js modules take precedence over the normal modules
+    // So, if a user creates a `constants` module and tries to require it
+    // with `import 'constants'`, the node.js module will be imported instead
+    //
+    // This configuration disables this behavior
+    // More information: https://github.com/webpack/webpack/issues/4666#issuecomment-292700060
+    node: {
+      constants: false
+    },
+
```

Comment:
I guess it is common practice to import local modules with relative paths. In the comments of the referring link they say that
`If you use const consts = require('./constants'); it will work.`
So I think sagui doesn't need to handle this. It is enough if users can add 
```
node: {
  constants: false
}
``` in their sagui.config.js

What do you think?